src/librustc_infer/infer/free_regions.rs

```diff
@@ -5,7 +5,6 @@
 
 use rustc_data_structures::transitive_relation::TransitiveRelation;
 use rustc_hir::def_id::DefId;
-use rustc_middle::middle::region;
 use rustc_middle::ty::{self, Lift, Region, TyCtxt};
 
 /// Combines a `region::ScopeTree` (which governs relationships between
```

Comment:
Missed reference to `region::ScopeTree`.

(don't mind me, I came across this PR from perf results, enjoying the diff)